Zotero/Scenes/Detail/DetailCoordinator.swift

```diff
@@ -241,14 +249,14 @@ final class DetailCoordinator: Coordinator {
         self.childCoordinators.append(coordinator)
         coordinator.start(animated: false)
 
-        self.navigationController.present(navigationController, animated: true, completion: nil)
+        self.navigationController?.present(navigationController, animated: true, completion: nil)
     }
 
     private func showWebView(for url: URL) {
         let controller = WebViewController(url: url)
         let navigationController = UINavigationController(rootViewController: controller)
         navigationController.modalPresentationStyle = .fullScreen
-        self.navigationController.present(navigationController, animated: true, completion: nil)
+        self.navigationController?.present(navigationController, animated: true, completion: nil)
```

Comment:
It will be more clear if the method starts with (or something like it)
```
        guard let currentNavigationController = self.navigationController else { return }
```
and then the local `navigationController` is initialized.